package.json

```diff
@@ -73,42 +73,42 @@
     "@bbc/gel-foundations": "6.1.3",
     "@bbc/moment-timezone-include": "1.1.8",
     "@bbc/psammead-assets": "3.1.9",
-    "@bbc/psammead-brand": "7.3.3",
-    "@bbc/psammead-bulleted-list": "3.1.0",
-    "@bbc/psammead-bulletin": "5.0.36",
-    "@bbc/psammead-byline": "3.0.19",
+    "@bbc/psammead-brand": "7.3.5",
+    "@bbc/psammead-bulleted-list": "3.1.2",
+    "@bbc/psammead-bulletin": "5.0.38",
+    "@bbc/psammead-byline": "3.0.21",
     "@bbc/psammead-calendars": "2.0.27",
-    "@bbc/psammead-caption": "4.1.14",
-    "@bbc/psammead-consent-banner": "5.5.3",
-    "@bbc/psammead-copyright": "3.0.19",
-    "@bbc/psammead-detokeniser": "1.0.6",
-    "@bbc/psammead-embed-error": "3.0.22",
+    "@bbc/psammead-caption": "4.1.16",
+    "@bbc/psammead-consent-banner": "5.5.5",
+    "@bbc/psammead-copyright": "3.0.21",
+    "@bbc/psammead-detokeniser": "1.0.7",
+    "@bbc/psammead-embed-error": "3.0.24",
     "@bbc/psammead-figure": "2.0.10",
-    "@bbc/psammead-grid": "3.1.2",
-    "@bbc/psammead-heading-index": "3.0.18",
-    "@bbc/psammead-headings": "5.0.18",
+    "@bbc/psammead-grid": "3.1.4",
+    "@bbc/psammead-heading-index": "3.0.20",
+    "@bbc/psammead-headings": "5.0.20",
     "@bbc/psammead-image": "2.0.8",
-    "@bbc/psammead-image-placeholder": "3.4.0",
-    "@bbc/psammead-inline-link": "3.0.16",
-    "@bbc/psammead-live-label": "2.0.21",
+    "@bbc/psammead-image-placeholder": "3.4.2",
+    "@bbc/psammead-inline-link": "3.0.18",
+    "@bbc/psammead-live-label": "2.0.23",
     "@bbc/psammead-locales": "5.0.7",
-    "@bbc/psammead-media-indicator": "6.1.2",
-    "@bbc/psammead-navigation": "9.2.2",
-    "@bbc/psammead-paragraph": "4.0.18",
-    "@bbc/psammead-play-button": "3.0.22",
+    "@bbc/psammead-media-indicator": "6.1.5",
+    "@bbc/psammead-navigation": "9.2.4",
+    "@bbc/psammead-paragraph": "4.0.20",
+    "@bbc/psammead-play-button": "3.0.24",
     "@bbc/psammead-rich-text-transforms": "2.0.6",
-    "@bbc/psammead-script-link": "3.0.20",
-    "@bbc/psammead-section-label": "7.1.0",
-    "@bbc/psammead-social-embed": "3.3.0",
-    "@bbc/psammead-story-promo": "8.0.21",
-    "@bbc/psammead-story-promo-list": "6.0.19",
-    "@bbc/psammead-styles": "7.3.0",
-    "@bbc/psammead-timestamp": "4.0.19",
-    "@bbc/psammead-timestamp-container": "5.0.30",
-    "@bbc/psammead-topic-tags": "1.0.0",
-    "@bbc/psammead-useful-links": "3.0.20",
+    "@bbc/psammead-script-link": "3.0.22",
+    "@bbc/psammead-section-label": "7.1.2",
+    "@bbc/psammead-social-embed": "3.3.2",
+    "@bbc/psammead-story-promo": "8.0.23",
+    "@bbc/psammead-story-promo-list": "6.0.21",
+    "@bbc/psammead-styles": "7.4.0",
+    "@bbc/psammead-timestamp": "4.0.21",
+    "@bbc/psammead-timestamp-container": "5.0.32",
+    "@bbc/psammead-topic-tags": "1.0.2",
+    "@bbc/psammead-useful-links": "3.0.22",
     "@bbc/psammead-visually-hidden-text": "2.0.7",
-    "@bbc/web-vitals": "1.0.8",
+    "@bbc/web-vitals": "1.0.9",
```

Comment:
Am I right in saying we still shouldn't bump `web-vitals` @jroebu14 ?

Reply:
Yep, needs to remain 1.0.8

Reply:
Sorry, out of the loop here. What is the issue with 1.0.9?

Reply:
I think that version caused an issue which was causing our pages to not render on Opera Mini Extreme mode, so it was pinned back.